lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission.feature

```diff
@@ -68,3 +68,9 @@ Feature: Accuracy of the Apellants Submission
     Given a user has completed the information needed on the accuracy of the appellant's submission page
     When the user returns to the submission accuracy page from the Task List
     Then the information they previously entered is still populated
+
```

Comment:
(BDD correctness, so i guess optional:
we shouldn't be making reference to pages - note anti pattern under 'Intention Revealing' in the article written by the guys who invented BDD.. https://cucumber.io/blog/bdd/keep-your-scenarios-brief/ )

Reply:
I think reviewing existing code is outside the remit of this code review? All I did was formatting